### DISCUSSION TITLE
Communication channel for MapLibre Flutter maintainers and contributors

### DISCUSSION THREAD

**Discussion by louwers** [maplibre/flutter-maplibre-gl#269] [General]
@m0nac0 @JulianBissekkou @stefanschaller (Please ping anyone else I forgot to mention.)

What is your preferred way to discuss? E-mail? Or Slack? Do you want to have a monthly meeting? Should we create a`#maplibre-flutter` channel on Slack?

Let me know if I can facilitate anything!

For now I suggest that when an important decision needs to be made, a thread on GitHub Discussions is made and then give the other maintainers and contributors a few days to respond before making the decision.

**Comment by JulianBissekkou** [comment]
Channel in the maplibre slack would be my prefered way of communication 👍🏽

**Reply by JulianBissekkou** [reply]
What about you @m0nac0 ?

**Reply by m0nac0** [reply]
Sounds good, I just joined the slack channel

**Comment by stefanschaller** [comment]
Slack and a new channel just for Flutter would be my preferred way. 🚀

**Comment by louwers** [comment]
Seems like this can be closed. 👍
